linode/manager
# upcoming: [M3-7874] - Linode Create Refactor - Marketplace App Sections

## Description 📝

- Adds the `New apps`, `Popular apps`, and `all apps` sections to the new Linode Create flow 🆕 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-05-28 at 3 28 15 PM](https://github.com/linode/manager/assets/115251059/e253f7c6-4f75-4390-a543-7b2dad5c855f) | ![Screenshot 2024-05-28 at 3 28 03 PM](https://github.com/linode/manager/assets/115251059/39f49dff-e1b6-49f3-9c47-b3853760eef9) |
| Notice how there are no sections  | It now looks like the old flow with the proper sections  | 

## How to test 🧪

### Prerequisites
- Turn on the Linode Create v2 feature flag using local dev tools

### Verification steps
- Checkout this PR locally or use the internal preview link
- Go to the Linode Create flow
- Got to the marketplace tab
- Verify you see the sections `New apps`, `Popular apps`, and `all apps` 
- Check for parity with production (the old Linode Create flow)
  - Verify the apps are the same under each section

## As an Author I have considered 🤔
